e2etest/pkg/frr/bgp.go

```diff
@@ -0,0 +1,40 @@
+package frr
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"go.universe.tf/metallb/e2etest/pkg/container"
+)
+
+// TODO: Leaving this package "test unaware" on purpouse, since we may find it
```

Comment:
s/purpouse/purpose

Also, maybe we could change the package name to make it less general (i.e. "frr")?

Reply:
If we want to make it general, we could connect to `bgpd` over the tcp/ip socket and pass the command. In this way, we wouldn't need to call `vtysh`.

Reply:
The package name IS `frr`, not sure you are suggesting a less general name than frr or you mis-interpreted the file name as the package name. If the former, I think frr is a good way to collect all the frr interaction utilities.

Reply:
Re: the socket, I'd tackle this separately, when the conversation around it is settled and when we decide to move this part of the code elsewhere. It could be that we find only the parsing part interesting, or that Ori throws everything away.

Reply:
i.e. "frr" meant the frr package. Yes, I think it should be less general.

Reply:
otherwise I can split and have `bgpneighbour.ForContainer` and such

Reply:
I guess it depends if you are going to eventually pull this into the main code base. Maybe its ok as long as the package is a sub-package of the test framework?

Reply:
If / when we move this to the main codebase we may need to rethink the package structure. I think we'll be able to recycle only the parsing part leaving the "execute in a container" side here.
So if I got it right, we leave frr for now, and we think about a more focused name when we move under metallb (I bet there'll be an frr package somewhere anyhow :-) )

Reply:
Yeah sounds like a plan